src/main/java/uk/gov/hmcts/reform/prl/models/dto/ccd/CaseData.java

```diff
@@ -786,4 +786,6 @@ public CaseData setIssueDate() {
     private final List<Element<HearingData>> listWithoutNoticeHearingDetails;
     private NextHearingDetails nextHearingDetails;
 
+    private final YesOrNo isAddCaseNumberAdded;
```

Comment:
We got a solution for adding new fields here. eventually we will end up having more than 255 attributes. FYI please.